### PR TITLE
[BD-24][BB-4204] Flag cleanup and minor fixes

### PR DESCRIPTION
**Pull request by giovannicimolin**
This PR simplifies the name of the LTI fields: they are now matching names from other LMS's. 
It also enables LTI 1.3 and Advantage features and removes unnecessary feature flags. 

Reviewers:
- [ ] @shimulch 
- [ ] @nedbat

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [BLENDED-880](https://openedx.atlassian.net/browse/BLENDED-880) to keep track of it in Jira.   More details are on the [BD-24](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1545076784/BD-24+LTI+v1.3+Improvements+Advantage+Support) project page.


When this pull request is ready, tag your edX technical lead.

**Comment by giovannicimolin** [comment]
@shimulch @nedbat Minor wording improvements, this is ready for review.

File: lti_consumer/utils.py [hunk 1]
```diff
@@ -16,21 +16,21 @@ def lti_1p3_enabled():
...
-    return settings.FEATURES.get('LTI_DEEP_LINKING_ENABLED', False) is True  # pragma: no cover
+    return settings.FEATURES.get('LTI_DEEP_LINKING_ENABLED', True) is True  # pragma: no cover
 
 
 def lti_nrps_enabled():
     """
     Returns `true` if LTI NRPS is enabled for instance.
     """
-    return settings.FEATURES.get('LTI_NRPS_ENABLED', False)  # pragma: no cover
+    return settings.FEATURES.get('LTI_NRPS_ENABLED', True)  # pragma: no cover
```
Comment (nedbat) [inline lti_consumer/utils.py:33]: We need to coordinate this with a change to edx.org settings to keep this disabled.

File: setup.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def is_requirement(line):
 
 setup(
     name='lti-consumer-xblock',
-    version='2.9.1',
+    version='2.9.2',
```
Comment (nedbat) [inline setup.py:52]: Please add an entry to the history in the readme.

**Review by nedbat — CHANGES_REQUESTED** [review]
One small change needed.

File: setup.py (hunk 2 above)
Comment (giovannicimolin) [reply setup.py:52]: Done!

File: lti_consumer/utils.py (hunk 1 above)
Comment (giovannicimolin) [reply lti_consumer/utils.py:33]: Makes sense (I can't review edX-internal PR's though - so I won't be able to help with this).

**Comment by giovannicimolin** [comment]
@nedbat Changes addressed and checks green :)

File: README.rst [hunk 3]
```diff
@@ -332,6 +332,14 @@ Please do not report security issues in public. Send security concerns via email
 Changelog
 =========
 
+2.9.2 - 2021-06-06
+------------------
+
+* LTI 1.3, alogin with LTI Advantage features is now enabled by default.
```
Comment (nedbat) [inline README.rst:338]:
```suggestion
* LTI 1.3, along with LTI Advantage features, is now enabled by default.
```

File: lti_consumer/utils.py (hunk 1 above)
Comment (nedbat) [reply lti_consumer/utils.py:33]: Right, and it doesn't have to happen before merging this, just before tagging to make a new release on PyPI.

Comment (giovannicimolin) [reply lti_consumer/utils.py]: @nedbat Is there need to gate this feature? The default configuration doesn't transmit any PII.

File: README.rst [hunk 4]
```diff
@@ -241,19 +233,13 @@ allow tools to send back grades. There's two grade interaction models implemente
...
 
-To enable LTI-DL and its capabilities, you need to set the following feature flag:
-
-.. code:: yaml
-
-    FEATURES:
-        LTI_DEEP_LINKING_ENABLED: true
-
-To enable LTI-NRP, you need to set the following feature flag:
+LTI-DL and its capabilities, you need to set these settings in the block:
```
Comment (nedbat) [inline README.rst:236]:
```suggestion
To enable LTI-DL and its capabilities, you need to set these settings in the block:
```

File: README.rst [hunk 5]
```diff
@@ -332,6 +318,14 @@ Please do not report security issues in public. Send security concerns via email
 Changelog
 =========
 
+2.9.2 - 2021-06-06
```
Comment (nedbat) [inline README.rst:321]:
```suggestion
2.10.0 - 2021-06-09
```

File: setup.py (hunk 2 above)
Comment (nedbat) [inline setup.py:52]:
```suggestion
    version='2.10.0',
```

**Review by nedbat — CHANGES_REQUESTED** [review]
Take the three tweaks, and then we can merge!

**Pull request merged**